modules/utxo-lib/test/fixtures_thirdparty/nodes/dogecoin-1.14.5/src/test/data/tx_valid.json

```diff
@@ -0,0 +1,514 @@
+[
+["The following are deserialized transactions which are valid."],
```

Comment:
same as before, how did you generate this one?

Reply:
It's also from [dogecoin github repo](https://github.com/dogecoin/dogecoin/blob/master/src/test/data/tx_valid.json)